frontend/src/lib/components/SaveToCohortModalContent/saveToCohortModalContentLogic.ts

```diff
@@ -0,0 +1,123 @@
+import { actions, afterMount, connect, kea, listeners, path, reducers, selectors } from 'kea'
+import { loaders } from 'kea-loaders'
+import { router } from 'kea-router'
+
+import { PaginationManual, lemonToast } from '@posthog/lemon-ui'
+
+import api, { CountedPaginatedResponse } from 'lib/api'
+import { delay } from 'lib/utils'
+import { cohortEditLogic } from 'scenes/cohorts/cohortEditLogic'
+import { projectLogic } from 'scenes/projectLogic'
+import { urls } from 'scenes/urls'
+
+import { ActorsQuery } from '~/queries/schema/schema-general'
+import { CohortType } from '~/types'
+
+import type { saveToCohortModalContentLogicType } from './saveToCohortModalContentLogicType'
+
+const COHORTS_PER_PAGE = 100
+
+export interface CohortFilters {
+    search?: string
+    page?: number
+    type: string
```

Comment:
We could be even more explicit here since there's only two possible values for the `type`, right?

```suggestion
    type: 'static' | 'dynamic'
```